docker-compose.yml

```diff
@@ -1,19 +1,70 @@
 services:
-  mysql:
+  mysql_auth:
     # Always specify what version, "latest" causes issues later on
     image: mysql:9.1
-    container_name: dev_db
     environment:
-      MYSQL_USER: ${MYSQLDB_USER}
-      MYSQL_PASSWORD: ${MYSQLDB_PASSWORD}
-      MYSQL_ROOT_PASSWORD: ${MYSQLDB_PASSWORD_ROOT}
-      MYSQL_DATABASE: ${MYSQLDB_NAME}
+#      MYSQL_USER: ${AUTH_MYSQL_USERNAME}
+      MYSQL_PASSWORD: ${AUTH_MYSQL_PASSWORD}
+      MYSQL_ROOT_PASSWORD: ${AUTH_MYSQL_ROOT_PASSWORD}
+      MYSQL_DATABASE: ${AUTH_DB_NAME}
     networks:
       - net
     ports:
-      - "${MYSQLDB_PORT}:3306"
+      - "${AUTH_DOCKER_DB_PORT}:${AUTH_DOCKER_DB_PORT}"
     volumes:
-      - ./dev/database:/var/lib/mysql
+      - ./dev/database/auth:/var/lib/mysql
+
+  eureka-service:
+    build:
+      context: ./Microservices/service-registry
+      dockerfile: Dockerfile
+    environment:
+      PORT_SERVICE_REGISTRY: ${PORT_SERVICE_REGISTRY}
+      SERVICE_REGISTRY_IP: eureka-service
+    networks:
+      - net
+    ports:
+      - "${PORT_SERVICE_REGISTRY}:${PORT_SERVICE_REGISTRY}"
+
+  gateway:
+    build:
+      context: ./Microservices/api-gateway
+      dockerfile: Dockerfile
+    environment:
+      GATEWAY_PORT: ${GATEWAY_PORT}
+      SERVICE_REGISTRY_IP: eureka-service
+      PORT_SERVICE_REGISTRY: ${PORT_SERVICE_REGISTRY}
+    networks:
+      - net
+    ports:
+      - "${GATEWAY_PORT}:${GATEWAY_PORT}"
+
+  auth_service:
+    build:
+      context: ./Microservices/IdentityMicroservice
+      dockerfile: Dockerfile
+    environment:
+      AUTH_MICROSERVICE_PORT: ${AUTH_MICROSERVICE_PORT}
+      AUTH_MYSQL_USERNAME: ${AUTH_MYSQL_USERNAME}
+      AUTH_MYSQL_PASSWORD: ${AUTH_MYSQL_ROOT_PASSWORD}
+      AUTH_MYSQL_ROOT_PASSWORD: ${AUTH_MYSQL_ROOT_PASSWORD}
+      AUTH_DB_IP: mysql_auth
+      AUTH_DB_PORT: ${AUTH_DB_PORT}
+      AUTH_DOCKER_DB_PORT: ${AUTH_DOCKER_DB_PORT}
+      AUTH_DB_NAME: ${AUTH_DB_NAME}
```

Comment:
Ye can pass in the entire ``.env`` file as shown here: https://github.com/Third-Floor-CSIS/cs4337-Big-Data-Group/blob/2e9640fbee6ef0e7e7b23cf47ecb9f40e8bbf250/docker-compose.yml#L29